src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -132,20 +136,21 @@ private BillableUsage produceMonthlyBillable(BillableUsage usage) {
     // Update the reported usage value to the newly calculated one.
     usage.setValue(usageCalc.getBillableValue());
 
-    // The orgId might not have been available when remittance
-    // was originally created, so we attempt to set here.
     if (updateRemittance(remittance, usage.getOrgId(), usageCalc)) {
+      remittance.setAccountNumber(
+          accountConfigRepository.findAccountNumberByOrgId(usage.getOrgId()));
```

Comment:
Instead of fetching the remittance, I'd rather just pass it along if present:

```suggestion
      remittance.setAccountNumber(usage.getAccountNumber());
```

Reply:
Done